juno-1/2300_v11_UPGRADE.md

```diff
@@ -0,0 +1,43 @@
+# Juno v11 Upgrade
+
+Juno v11 is a minor change that adds:
+
+- bug fixes
+- new ICA configuration
+- a pruning command
+- CosmWasm 1.1
+
+Important! you _MUST_ apply this config in `app.toml` when upgrading. This should go in the base configuration section, e.g. under `index-events`:
```

Comment:
I would say, "under the `index-events[]` configuration line.

Also should bold **Important!**